### ISSUE TITLE
spellforce.ru, 2007

### ISSUE THREAD

**Issue by igeler** [leszekd25/spellforce_data_editor#8]
Гутен таг! Пардон, что пишу сюда — твоя почта и ВК молчат.
Мы знакомы — общались в 2007-2009 и 2018 насчёт модинга SF.
Впечатлён твоим редактором, круто было бы обсудить его фичи.

**Comment by leszekd25** [comment]
Здравствуйте! К сожалению, я не могу правильно писать на русском языке, поэтому буду использовать Google Translate.
Я очень сомневаюсь, что мы говорили в 2007 году, так как у меня тогда не было Интернета. Кроме того, у меня нет учетной записи ВКонтакте. Я думаю, что вы ошибаетесь с кем-то : )

**Comment by igeler** [comment]
Excuse me! :3 I really confused you with [this person](https://github.com/Hokan-Ashir/SFGameDataEditor).
Do you use telegram?

**Comment by leszekd25** [comment]
Nope, but I use Discord a lot. Discord tag: **shovel_knight#7698** if you want to talk about SF stuff (thought i may or may not be available at this hour).

**Comment by igeler** [comment]
Thank you! I will write you today or tomorrow.
